src/cmd/health.js

```diff
@@ -218,18 +230,14 @@ class NSFSHealth {
             });
         } catch (err) {
             console.log('Error while pinging endpoint host :' + HOSTNAME + ', port ' + this.https_port, err);
-            return {
-                response: fork_response_code.NOT_RUNNING.response_code,
-            };
+            endpoint_state = { response: fork_response_code.NOT_RUNNING };
         }
         return endpoint_state;
     }
 
-    async get_error_code(nsfs_status, pid, rsyslog_status, endpoint_response_code) {
+    async get_error_code(nsfs_status, pid, endpoint_response_code) {
         if (nsfs_status !== 'active' || pid === '0') {
             return health_errors.NOOBAA_SERVICE_FAILED;
-        } else if (rsyslog_status !== 'active') {
```

Comment:
Are we not using the error code `RSYSLOG_SERVICE_FAILED` for rsyslog?

Reply:
I had a short discussion with @guymguym regarding the checks here of rsyslog/syslog_ng, after this discussion my suggestion is to have both checks for rsyslog/syslog_ng be optional and that the overall health should be affected by the status of these services. 
As mentioned in the description - service health is no longer affected by rsyslog/syslog_ng status, even if they are failing the overall status will be OK, their errors will be reflected in the rsyslog/syslog_ng service section.

Reply:
ok, got it.